### DISCUSSION TITLE
WIll it work with Static IP Queue system?

### DISCUSSION THREAD

**Discussion by mdarparvez** [hotspotbilling/phpnuxbill#43] [Q&A]
Hello, this is great software you've developed.
In my ISP, we don't use the ppoe or hotpost system. We are providing service to our client with static IP queues in Mikrotik.
Will phpnuxbill work with queues static ip system?

**Comment by djpao29** [comment]
Consider adding support for the "Add client to simple queue" feature for static IPs to manage clients using simple queues, queue trees, and PCQ, not just PPPoE, within PHPNuxBill. This would allow bandwidth control via static IPs (/32) using maximum limits and other queuing methods, which is essential for many ISPs.
